cmake_modules/san-config.cmake

```diff
@@ -37,7 +37,7 @@ if (${TERRIER_USE_ASAN})
       message(SEND_ERROR "Cannot use dynamic linking when ASAN and UBSAN are both enabled")
     endif ()
   endif ()
-  set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -fsanitize=address")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
```

Comment:
Isn't `CXX_COMMON_FLAGS` already appended to `CMAKE_CXX_FLAGS`? This has the side effect of removing the flag from other compilation steps that use `CXX_COMMON_FLAGS`.

Reply:
That is a bug. Good catch. Here was my reasoning: its appended, but later in the main CMakeLists.txt. By the time it's appended, other steps have already passed, and many dependencies have already been configured _not_ to use a sanitizer.


However, this breaks from expected behavior for util's CMakeList. I'll fix that right now.